### ISSUE TITLE
[Request] More playback options

### ISSUE THREAD

**Issue by cjmwid** [flyth/ts3soundbot#389]
Maybe like a seek and restart. As i find myself using next then prev to start the song over

**Comment by SilverCory** [comment]
Agree, a playall and shuffle all would be great aswell. Over 10000 songs, all imported now but playlists can only have 1000 songs max. Just want to leave it to do it's thing!
